lib/storext/override.rb

```diff
@@ -89,6 +96,11 @@ def storext_overrider_define_override_control(column_name, attr)
           end
         end
 
+        define_method :discard_value? do
```

Comment:
This looks like it will create `discard_value?` for every attribute we have. It will keep overwriting the last one.